release_notes/ocp-4-6-release-notes.adoc

```diff
@@ -1022,6 +1022,8 @@ For more information, see xref:../monitoring/understanding-the-monitoring-stack.
 Red Hat does not guarantee backward compatibility for metrics, recording rules, or alerting rules.
 ====
 
+For more information about alerting rule changes and monitoring component version updates in {product-title} 4.6, see link:https://raw.githubusercontent.com/openshift/cluster-monitoring-operator/release-4.6/CHANGELOG.md[the `cluster-monitoring-operator` change log for `release-4.6`].
```

Comment:
@pneedle-rh This LGTM and the link works. It's a little more verbose, but what do you think about the following wording:

> For more information about changes to alerting rules and updates to the monitoring component version in {product-title} 4.6, see link:https://raw.githubusercontent.com/openshift/cluster-monitoring-operator/release-4.6/CHANGELOG.md[the `cluster-monitoring-operator` change log for `release-4.6`].

Reply:
@lbarbeevargas thank you for your review and for your suggestion. I have updated this line to incorporate your recommendation.

One change I made to the rewording is that I stated "to monitoring component versions" plural, because this relates to many of the components in the Monitoring stack.